var/spack/repos/builtin/packages/bamtools/package.py

```diff
@@ -36,11 +37,14 @@ class Bamtools(Package):
     version('2.3.0', 'd327df4ba037d6eb8beef65d7da75ebc')
     version('2.2.3', '6eccd3e45e4ba12a68daa3298998e76d')
 
-    depends_on('cmake', type='build')
+    depends_on('zlib', type='link')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            cmake('..', *std_cmake_args)
+    def cmake_args(self):
+        args = []
 
-            make()
-            make('install')
+        rpath = self.rpath
+        rpath.append(os.path.join(self.prefix.lib, "bamtools"))
+        args.extend(std_cmake_args)
+        args.append("-DCMAKE_INSTALL_RPATH=%s" % ':'.join(rpath))
```

Comment:
If you check out the `CMakePackage` base class:
```
$ spack edit -b cmake
```
you'll see that the `cmake()` function already adds `std_cmake_args` to the call, so this long list will actually be duplicated. It might be better to override `std_cmake_args` and find the `CMAKE_INSTALL_RPATH` argument and add to it directly.